### PR TITLE
Replace all downloadable file images currently in main with access date images

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
⚠️ Stacked on #180 – see that PR for the sentence included at the beginning of the downloadable files section. 

Here, I'm addressing the part of #171 concerned with download illustrations already in `main`: project, sample, multiplex, and spatial. Ultimately, I've chosen to get this into `main` via the `documentation-improvements` branch, which contrasts what I wrote on the issue initially (written before our discussion on Friday). We will eventually merge `documentation-improvements` -> `main` and then `main` -> `development`, so I figured there was no harm in this approach.

The most important thing to check is that I didn't save an image under the wrong name!

### CODE REVIEW

File: docs/images/multiplexed-download-folder.png
Comment (allyhawkins) [inline docs/images/multiplexed-download-folder.png:1]: I think we might need to update this image. We have `processed.rds` files that are available for multiplexed data that is not reflected here. Looks like we missed updating that image.

**Review by allyhawkins — COMMENTED** [review]
Everything looks fine, but I did notice that we are missing a file that should be in the multiplexed image. I think we should make sure to update that so that each sample folder also has `SCPCL000000_processed.rds`. Tagging @dvenprasad for help with that.

**Comment by dvenprasad** [comment]
Here is the updated multiplexed download folder
multiplexed-download-folder

File: docs/images/multiplexed-download-folder.png
Comment (jaclyn-taroni) [reply docs/images/multiplexed-download-folder.png:1]: Updated in https://github.com/AlexsLemonade/scpca-docs/pull/181/commits/66562f16b5c26f172379ead8f8aa4c991d197efe

**Review by allyhawkins — APPROVED** [review]
Looks good!

**Pull request merged**